src/auth/user.ts

```diff
@@ -34,3 +34,32 @@ export interface User extends ArenaObject<UserProps> {
   prefs?: UserPrefs
   status: UserStatus
 }
+
+
+// Methods
+
+const getAuthGroups = (user:User):AuthGroup[] => user?.authGroups || []
+
+const isSystemAdmin =  (user:User):boolean => (getAuthGroups(user) || []).some(AuthGroupMethods.isSystemAdminGroup)
+
+const getAuthGroupBySurveyUuid= (surveyUuid: string | undefined, includeSystemAdmin = true) => (user: User):AuthGroup | undefined => {
+  const authGroups = getAuthGroups(user)
+  return includeSystemAdmin && isSystemAdmin(user) && authGroups ? authGroups[0] : authGroups?.find(group => group.surveyUuid === surveyUuid)  
+}
+
+// Getters
+const getUuid = (user:User): string => user.uuid
+
+// ====== CHECK ( Object Utils )
+const isEqual = (other: User) => (self: User): boolean => getUuid(other) === getUuid(self)
+const hasAccepted = (user: User): boolean => user.status === UserStatus.ACCEPTED
+
+export const UserMethods = {
+  isSystemAdmin,
+  getAuthGroupBySurveyUuid,
+  //Getters
+  getUuid,
+  //check
+  isEqual,
+  hasAccepted
+}
```

Comment:
let's create a separate file called Users that export those method (utils convention)